.travis.yml

```diff
@@ -29,6 +29,8 @@ env:
   - PROVIDER=softlayer
   - PROVIDER=triton
 
+install: echo "Skipping install step, everything is included in the docker image"
```

Comment:
I wanted it to be self explanatory, but I can see your point about how redundant it is. 
> On Jul 6, 2016, at 10:59, Langston Barrett <notifications@github.com> wrote:
> 
> In .travis.yml <https://github.com/CiscoCloud/mantl/pull/1649#discussion_r69759375>:
> 
> > @@ -29,6 +29,8 @@ env:
> >    - PROVIDER=softlayer
> >    - PROVIDER=triton
> >  
> > +install: echo "Skipping install step, everything is included in the docker image"
> Why not true?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/CiscoCloud/mantl/pull/1649/files/f4661b77d4b502427f671ef0ec3d150dbd78fcee#r69759375>, or mute the thread <https://github.com/notifications/unsubscribe/ACnp5BZ0Y4qJJsJRiG9hwjiqkQ4YrKT4ks5qS9DmgaJpZM4JGO-u>.
>